### ISSUE TITLE
An ability to set custom taints on Node startup

### ISSUE THREAD

**Issue by zuzzas** [deckhouse/deckhouse#2172]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

We'd like to delay scheduling until we clear a condition with a separate controller. We can't use node template, since it'll reinstate the taint immediately after its removal from the Node object.

### Proposed Solution

I'd like to have a field in a NodeGroup object that set taint on a Node object only once. Ideally passed via kubelet config or flags.

**Comment by EvgenySamoylov** [comment]
Looks like a nice feature, but out of scope right now.
